compiler/rustc_span/src/symbol.rs

```diff
@@ -1536,21 +1578,32 @@ pub struct Interner {
 }
 
 impl Interner {
-    fn prefill(init: &[&'static str]) -> Self {
-        Interner {
-            strings: init.into(),
-            names: init.iter().copied().zip((0..).map(Symbol::new)).collect(),
-            ..Default::default()
-        }
+    pub fn fresh() -> Self {
```

Comment:
`Interner::default` already exists, so this method can be removed.

Reply:
Removed.